tools/build_pytorch_libs.py

```diff
@@ -61,47 +58,16 @@ def build_caffe2(version,
                  build_dir):
     my_env = _create_build_env()
     build_test = not check_negative_env_flag('BUILD_TEST')
-    max_jobs = os.getenv('MAX_JOBS', str(multiprocessing.cpu_count()))
-    cmake_cache_file = os.path.join(build_dir, 'CMakeCache.txt')
-    ninja_build_file = os.path.join(build_dir, 'build.ninja')
-    if rerun_cmake and os.path.isfile(cmake_cache_file):
-        os.remove(cmake_cache_file)
-    if not os.path.exists(cmake_cache_file) or (
-            USE_NINJA and not os.path.exists(ninja_build_file)):
-        cmake.run(version,
-                  cmake_python_library,
-                  build_python,
-                  build_test,
-                  build_dir,
-                  my_env)
+    c = CMake(build_dir)
```

Comment:
nit: use a better name here

Reply:
I fixed all the other comments. Do you think `CMakeManager` is better?

Reply:
No, I mean the name of the variable `c`. Would you please try to make it a bit more meaningful?

Reply:
Indeed, can't believe I gave such a name. Updated.